custom_components/flashforge_adventurer_3/strings.json

```
@@ -3,12 +3,13 @@
     "step": {
       "user": {
         "description": "Please enter your printer's IP address and port.",
-        "title": "FlashForge Adventurer 3 setup",
+        "title": "FlashForge Printer Setup",
         "data": {
+          "model": "Printer Model (Adventurer 3, Adventurer 5M Pro, etc.)",
```

```suggestion
          "model": "Printer model",
```

given that it'll display all supported models underneath.